packages/@aws-cdk/aws-timestream/lib/table.ts

```diff
@@ -0,0 +1,223 @@
+import { UnknownPrincipal } from '@aws-cdk/aws-iam';
+import { ArnFormat, Duration, IResource, Resource, Stack } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { IDatabase } from './database';
+import { EncryptionOptions } from './enums';
+import { CfnTable, CfnTableProps } from './timestream.generated';
+
+/**
+ * Table Interface
+ */
+export interface ITable extends IResource {
+  /**
+   * Table ARN
+   *
+   * @attribute
+   */
+  readonly tableArn: string;
+
+  /**
+   * Table Name
+   *
+   * @attribute
+   */
+  readonly tableName: string;
+
+  /**
+   * Database Name
+   */
+  readonly databaseName: string
+}
+
+/**
+ * S3 Configuration
+ */
+export interface MagneticS3Configuration {
+  /**
+   * The name of the S3 bucket.
+   */
+  readonly bucketName: string,
+
+  /**
+   * The encryption option for the S3 location. Valid values are S3 server-side encryption with an S3 managed key (SSE_S3) or AWS managed key (SSE_KMS).
+   */
+  readonly encryptionOption: EncryptionOptions,
```

Comment:
Should this be optional?

Reply:
What would a sensible default look like here?

Reply:
I'm thinking if this is going to be set a default SSE_S3 seems reasonable?